.travis.yml

```diff
@@ -52,9 +52,7 @@ install:
 script:
   - conda activate $TEST_ENV
   - echo `which python`
-  - echo `which nosetests`
-  - |
-    cd ${TEST_ENV_PREFIX}/ilastik-meta/lazyflow/tests && nosetests --nologcapture --ignore-files=testInterpolatedFeatures.py \
-    --with-coverage --cover-xml \
-    --cover-inclusive --cover-package=lazyflow .
-  - coveralls
+  - echo `which pytest`
```

Comment:
i guess this is some debug-leftover?

Reply:
No, just to be sure that it uses conda env pytest